wasm2c/examples/rot13/main.c

```diff
@@ -19,48 +19,69 @@
 
 #include "rot13.h"
 
-/* Define the imports as declared in rot13.h. */
-wasm_rt_memory_t (*Z_hostZ_mem);
-u32 (*Z_hostZ_fill_buf)(u32, u32);
-void (*Z_hostZ_buf_done)(u32, u32);
+/* Define structure to hold the imports */
+struct Z_host_module_instance_t {
+  wasm_rt_memory_t s_memory;
+  char* s_input;
```

Comment:
These members probably shouldn't have the `s_` prefix if they are not static (i think `s_` is for static here)

Reply:
Ah, fixed.